openSUSE/open-build-service
# Use component to delete all built binaries

Use DeleteConfirmationDialogComponent instead of a partial to render the confirmation dialog.

How to test this:
- make a package build for many architectures of the same repository
- click on the repository name link in the build results (or either go to `/package/binaries/<project>/<package>/<repository>`)
- click on the "Delete all built binaries" action on the top of the page
- all the binaries of **all** the architectures should disappear
